lib/gui/pages/main/styles/_main.scss

```diff
@@ -110,11 +89,21 @@
 
 .page-main .button.step-footer {
   font-size: 12px;
-  border-bottom: 1px dashed;
+  color: $palette-theme-primary-background;
   border-radius: 0;
   padding: 0;
 }
 
 .page-main .glyphicon {
   vertical-align: text-top;
 }
+
+.step-drive-name,
```

Comment:
Given these are the same thing, can we make the class names more generic (e.g: not tied to the concept of a drive/image to we can re-use them)?

For example: `.step-name` and `.step-description`?